documents/openapi/openapi.yaml

```diff
@@ -2512,6 +2547,21 @@ components:
       required:
         - recipient_identifier
         - template_id
+    PushNotificationBroadcastRequest:
+      type: object
+      properties:
+        mobile_app:
+          $ref: "#/components/schemas/MobileApp"
+        template_id:
+          $ref: "#/components/schemas/Id"
+        personalisation:
+          $ref: "#/components/schemas/Personalisation"
```

Comment:
Might need to just add a new schema with 
```
{    
    "Message", "New update for the Mobile App..."  
   }  
```

Reply:
done